configure.ac

```diff
@@ -1516,6 +1515,9 @@ AC_CHECK_HEADERS([sys/types.h \
                   editline/readline.h \
                   ucred.h ])
 
+# On OpenBSD, pthread.h must be included before pthread_np.h. #1306
```

Comment:
I don't propose doing it in general, just in this case I don't have another citation for this claim about OpenBSD. If it isn't true for every instance of OpenBSD, someone might question why this is in there, and the issue explains that. There's always "blame", this is just less hassle for that person, I think. There are cases where we reference Jiras from comments, no?